src/hotspot/cpu/zero/zeroInterpreter_zero.cpp

```diff
@@ -551,8 +551,9 @@ int ZeroInterpreter::native_entry(Method* method, intptr_t UNUSED, TRAPS) {
 }
 
 int ZeroInterpreter::getter_entry(Method* method, intptr_t UNUSED, TRAPS) {
+  JavaThread* thread = THREAD->as_Java_thread();
```

Comment:
This will be unnecessary after my TRAPS change but that is still a little ways off. :)